source/Engine.cpp

```diff
@@ -837,8 +837,18 @@ void Engine::Step(bool isActive)
 		targetSwizzle = target->GetSwizzle();
 		info.SetString("mission target", target->GetPersonality().IsTarget() ? "(mission target)" : "");
 
+		// Only update the "active" state shown for the target if it is
+		// in the current system and targetable, or owned by the player.
 		int targetType = RadarType(*target, step);
-		info.SetOutlineColor(GetTargetOutlineColor(targetType));
+		if(targetType == Radar::BLINK)
+			info.SetOutlineColor(GetTargetOutlineColor(targetType));
```

Comment:
Lines 848 and 850 both perform `info.SetOutlineColor`. Is it still needed to also perform the call here?

Reply:
You are correct. This whole thing is wrong.